src/components/DocumentationTopic/TopicsTable.vue

```diff
@@ -27,21 +27,31 @@
       <template v-if="section.discussion" slot="discussion">
         <ContentNode :content="section.discussion.content" />
       </template>
-      <TopicsLinkBlock
-        v-for="topic in section.topics"
+      <TopicsLinkCardGrid
+        v-if="!shouldRenderList"
```

Comment:
You can totally ignore this if you want, but I always find the pattern `if not condition {} else {}` to be slightly harder to follow in cases where it could be reframed as `if condition {} else {}`

If you want, you could either:
* change the flag (and its computation) to be `shouldRenderGrid` and the condition could drop the `!` here
* keep `shouldRenderList` as-is and just swap the order of the conditional so it's `if shouldRenderList {} else {}`

Also, feel free to ignore this entirely—just a minor style thing. :P